### DISCUSSION TITLE
How can I migrate my custom modules from Vue 2 to Vue 3

### DISCUSSION THREAD

**Discussion by sarthakgaur** [directus/directus#7154] [Archive]
Can someone list the steps that you'll have to perform to migrate your custom modules from Vue 2 to Vue 3?

**Comment by nickrum** [comment]
1. Make sure all dependencies you are using are compatible with Vue 3.
2. Take a look at the [module docs](https://docs.directus.io/guides/modules/) and look out for changes. In case of modules, only the `routes` field has changed, requiring you to remove the leading slash from route paths.
3. Consult the Vue 3 [migration guide](https://v3.vuejs.org/guide/migration/introduction.html) and adjust your extension accordingly.
